src/bacnet/bacapp.c

```diff
@@ -1404,50 +1430,67 @@ bool bacapp_parse_application_data(BACNET_APPLICATION_TAG tag_number,
         status = true;
         value->tag = tag_number;
         switch (tag_number) {
+#if defined(BACAPP_BOOLEAN)
             case BACNET_APPLICATION_TAG_BOOLEAN:
                 long_value = strtol(argv, NULL, 0);
                 if (long_value)
                     value->type.Boolean = true;
                 else
                     value->type.Boolean = false;
                 break;
+#endif
+#if defined(BACAPP_UNSINED)
```

Comment:
misspelled define:  `BACAPP_UNSINED` should be `BACAPP_UNSIGNED` and is causing failure in unit test.

Reply:
thanks